modules/@apostrophecms/ui/ui/apos/scss/_inputs.scss

```diff
@@ -61,7 +61,8 @@ $input-max-width: 580px;
   &--time,
   &--textarea,
   &--date,
-  &--select {
+  &--select,
+  &--password {
```

Comment:
this is getting gnarly but not this pr's problem ...

Reply:
true